examples/frameworks/nuxt-4-caching/app/app.vue

```diff
@@ -0,0 +1,108 @@
+<template>
+  <NuxtRouteAnnouncer />
+
+  <header>
+    <h1>Nuxt 4 on Netlify</h1>
+
+    <p class="subheading">A simple Nuxt 4 app deployed on Netlify</p>
+  </header>
+
+  <main>
+    <NuxtPage />
```

Comment:
If you wrap the pages in a `<section>` (either here or in the page), you'll get some nicer auto-formatting.